woocommerce/woocommerce-android
# Issue/2985 ul label fix

Closes #2985 by updating the label used on the login email screen to match the design. 

Before | After 
-- | --
![Screenshot_1603986256](https://user-images.githubusercontent.com/5810477/97598221-18d4d400-19dd-11eb-9d4e-9d6e37ef1288.png)|![Screenshot_1603986579](https://user-images.githubusercontent.com/5810477/97598226-1a060100-19dd-11eb-80de-a3e1c92daffb.png)

### To Test
1. Start app logged out
2. Tap "Continue with WordPress.com"
3. Verify the label text is correct.

Update release notes:
